addon/ng2/blueprints/component/index.js

```diff
@@ -4,7 +4,7 @@ var Blueprint = require('ember-cli/lib/models/blueprint');
 var dynamicPathParser = require('../../utilities/dynamic-path-parser');
 var addBarrelRegistration = require('../../utilities/barrel-management');
 var getFiles = Blueprint.prototype.files;
-const stringUtils = require('ember-cli/lib/utilities/string');
+const stringUtils = require('ember-cli-string-utils');
```

Comment:
`ember-cli` with version `2.5.0` does not comes with included string utils, looks like they have removed it from `ember-cli` and added separated package.